web/src/app/projects/repository/repository.component.html

```diff
@@ -15,7 +15,7 @@
 </div>
 <mat-accordion class="repository" *ngIf="repository">
   <mat-expansion-panel hideToggle #expansion [expanded]="true">
-    <mat-expansion-panel-header [collapsedHeight]="headerHeight+'px'" [expandedHeight]="headerHeight+'px'"
+    <mat-expansion-panel-header [collapsedHeight]="headerHeight+'px'" [expandedHeight]="headerHeight+'px'" 
```

Comment:
Extra space at end of the line is not needed? There are few of these below

Reply:
Actually when I did Alt+Shift+F , it has automatically formatted the files and moved the long lines in new lines . When I checked the difference in VS Code , I fixed the difference and I couldn't see the extra spacing diff in my VS code. So it occurred.

Reply:
Oh ok 🤔